.prettierignore

```diff
@@ -0,0 +1,58 @@
+/_test/
+/public/build/
+.github/workflows/ci.yml
+CHANGELOG.md
+public/global.css
+public/index.html
+fetch-fixtures.js
+generate-color-css.js
+public/service-worker.js
+rollup.config.js
+src/App.svelte
+src/Catalog.svelte
+src/ErrorBoundary.js
+src/ErrorBoundary.svelte
+src/SearchResults.svelte
+src/Thing.svelte
+src/all.test.ts
+src/createBoundary.ts
+src/data.ts
+src/main.ts
+src/schema.test.ts
+src/types.ts
+src/types/AmmunitionType.svelte
+src/types/AsciiPicture.svelte
+src/types/Construction.svelte
+src/types/Fault.svelte
+src/types/Flag.svelte
+src/types/Furniture.svelte
+src/types/Item.svelte
+src/types/Material.svelte
+src/types/Monster.svelte
+src/types/Proficiency.svelte
+src/types/Recipe.svelte
+src/types/Skill.svelte
+src/types/Technique.svelte
+src/types/ThingLink.svelte
+src/types/ToolQuality.svelte
+src/types/Unknown.svelte
+src/types/VehiclePart.svelte
+src/types/Vitamin.svelte
+src/types/item/AmmoInfo.svelte
+src/types/item/ArmorInfo.svelte
+src/types/item/BookInfo.svelte
+src/types/item/ComestibleInfo.svelte
+src/types/item/ComponentOf.svelte
+src/types/item/Deconstruct.svelte
+src/types/item/Disassembly.svelte
+src/types/item/DroppedBy.svelte
+src/types/item/FurnitureSymbol.svelte
+src/types/item/GrownFrom.svelte
+src/types/item/GunInfo.svelte
+src/types/item/ItemSymbol.svelte
+src/types/item/MeleeInfo.svelte
+src/types/item/Recipes.svelte
+src/types/item/RequirementData.svelte
+src/types/item/SpawnedIn.svelte
+src/types/item/WheelInfo.svelte
+svelte.config.js
```

Comment:
You should be able to drop the prettierignore now!

Reply:
already did